db/migrate/20200625153607_create_notes.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateNotes < ActiveRecord::Migration[6.0]
+  def change
+    create_table :notes do |t|
+      t.string :created_by
```

Comment:
I'd use the actual primary key of the `users` table to represent relations or drop the primary key column entirely in favor of the `gravity_user_id`.

Reply:
Good points. I will reference the `gravity_user_id` field.